### PR TITLE
Upgrade plugin to support new sylius

### PR DESCRIPTION
**Pull request by andriusvo**

### CODE REVIEW

**Comment by andriusvo** [comment]
@bigboss86

File: composer.json [hunk 1]
```diff
@@ -9,8 +9,8 @@
         "friendsofsymfony/ckeditor-bundle": "^1.1 || ^2.1",
         "knplabs/knp-gaufrette-bundle": "^0.5 || ^0.7",
         "sylius/grid-bundle": "^1.6",
-        "sylius/resource-bundle": "^1.4",
-        "symfony/framework-bundle": "^4.4 || ^5.2",
+        "sylius/resource-bundle": "^1.9",
+        "symfony/framework-bundle": "^4.4 || ^5.2 | ^6.0",
```
Comment (bigboss86) [inline composer.json:13]:
```suggestion
        "symfony/framework-bundle": "^4.4 || ^5.2 || ^6.0",
```

File: composer.json [hunk 2]
```diff
@@ -36,11 +36,11 @@
...
-        "symfony/browser-kit": "^4.4 || ^5.2",
-        "symfony/debug-bundle": "^4.4 || ^5.2",
-        "symfony/dotenv": "^4.4 || ^5.2",
-        "symfony/intl": "^4.4 || ^5.2",
-        "symfony/web-profiler-bundle": "^4.4 || ^5.2",
+        "symfony/browser-kit": "^4.4 || ^5.2 | ^6.0",
+        "symfony/debug-bundle": "^4.4 || ^5.2 | ^6.0",
+        "symfony/dotenv": "^4.4 || ^5.2 | ^6.0",
+        "symfony/intl": "^4.4 || ^5.2 | ^6.0",
+        "symfony/web-profiler-bundle": "^4.4 || ^5.2 | ^6.0",
```
Comment (bigboss86) [inline composer.json:43]:
```suggestion
        "symfony/browser-kit": "^4.4 || ^5.2 || ^6.0",
        "symfony/debug-bundle": "^4.4 || ^5.2 || ^6.0",
        "symfony/dotenv": "^4.4 || ^5.2 || ^6.0",
        "symfony/intl": "^4.4 || ^5.2 || ^6.0",
        "symfony/web-profiler-bundle": "^4.4 || ^5.2 || ^6.0",
```

**Review by bigboss86 — APPROVED** [review]

**Comment by bigboss86** [comment]
Thanks @andriusvo

**Pull request merged**